ts/src/coder/index.ts

```diff
@@ -28,6 +28,11 @@ export interface Coder {
    * Coder for events.
    */
   readonly events: EventCoder;
+
+  /**
+   * Coder for user-defined types.
+   */
+  readonly types: TypesCoder<T>;
```

Comment:
Please add a CHANGELOG.md entry for this change.

Reply:
Done